src/components/auth/LoginForm.scss

```diff
@@ -12,13 +12,31 @@
     color: lightslategray;
   }
 
-  button {
-    background-color: green;
-    margin-top: 1.5em;
-    font-weight: bold;
-    margin-left: auto;
-    margin-right: auto;
-    padding: 1rem 2rem;
-    border-radius: .3rem;
+  &__buttons {
+    display: flex;
+    margin-top: 16px;
+
+    button {
+      display: inline-block;
+      background-color: green;
+      width: 48%;
+      font-weight: bold;
+      padding: 1em 2em;
+      border-radius: .3em;
+    }
+
+    .google-login {
```

Comment:
Don't use 'global' classes. Please make unique class like LoginForm__google-login and reference it like
```
.LoginForm {
  &__google-login {
    ...
  }
}
```